packages/app-builder-lib/src/util/yarn.ts

```diff
@@ -146,16 +133,11 @@ export async function nodeGypRebuild(platform: NodeJS.Platform, arch: string, fr
 function getPackageToolPath(pm: PM) {
   let cmd = pm
   if (process.env.FORCE_YARN === "true") {
-    cmd = "yarn"
+    cmd = PM.YARN
   }
   return `${cmd}${process.platform === "win32" ? ".cmd" : ""}`
```

Comment:
Just to double check. Does this still work if `PM.YARN_BERRY` is used here? Is the `cmd` on Windows `yarn-berry.exe`?

Reply:
No, it's still `yarn.cmd`.

![image](https://github.com/user-attachments/assets/6cac5325-5ae9-42f0-8982-e26241c7a5a2)

Reply:
@mmaietta  Sorry for the misunderstanding. Yarn Berry still uses `yarn.cmd`, not `yarn-berry.cmd`. This has already been verified.

Reply:
This logic here still doesn't seem right to me.

```
function getPackageToolPath(pm: PM) {
  let cmd = pm
  if (process.env.FORCE_YARN === "true") {
    cmd = "yarn"
    cmd = PM.YARN
  }
  return `${cmd}${process.platform === "win32" ? ".cmd" : ""}`
}
```
When `pm = PM.YARN_BERRY && platform === win32`, it returns -> `return "yarn-berry.cmd"`

Reply:
My bad, I misunderstood. I’ve added this function to `packageManager`, so others can directly call it when needed, without having to create their own.